### PR TITLE
Grant: Substrate Tokenomics Survey

### PR DESCRIPTION
**Pull request by taqtiqa-mark**
#### Project Abstract

A survey of some economic models used to develop Substrate (Dotsama) blockchain/token economies.
The published literature will be from the rational expectations equilibrium (a.k.a. no-arbitrage) approach to economic analysis.

#### Grant [level](https://github.com/w3f/Grants-Program#level_slider-levels)

- [x] **Level 1**:  Up to $10,000, 2 approvals
- [ ] **Level 2**:  Up to $30,000, 3 approvals
- [ ] **Level 3**:  Unlimited, 5 approvals (for >$100k: Web3 Foundation Council approval)

#### Application Checklist

- [x] The [application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md) has been copied and aptly renamed (`project_name.md`).
- [x] I have read the [application guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/grant_guidelines_per_category.md).
- [ ] A BTC, Ethereum (USDC/DAI) or Polkadot/Kusama (aUSD/USDT) address for the payment of the milestones is provided in the application.
- [x] The software delivered for this grant will be released under an open-source license specified in the application.
- [x] The initial PR contains only one commit (squash and force-push if needed).
- [x] The grant will only be announced once the first milestone [has been accepted](https://github.com/w3f/Grant-Milestone-Delivery#process) (see the [announcement guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/announcement-guidelines.md)).

### CODE REVIEW

**Review by Noc2 — CHANGES_REQUESTED** [review]
Thanks for the application. Feel free to remove all the parts of the template that are not relevant to a research application. We need to come up with a different template for research-focused applications. This way, it will be easier to understand. 
Apart from this, I will share your application with a few additional people for more feedback. One initial feedback from me is that the application is quite expensive, and it might be easier to initially apply for a smaller grant so that we can get a better understanding of your work.

**Comment by laboon** [comment]
I think this would be a valuable addition to the space, but the price does seem rather expensive. 

This was filed as a Level 3 grant (> $100k), but I only see three milestones, each for $18k (= $54k). Perhaps there is something I am missing here or it was simply a typo?

**Review by Noc2 — CHANGES_REQUESTED** [review]
Also, could you add the overview section: https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md#overview-1 ?

**Comment by taqtiqa-mark** [comment]
@Noc2:

> add the overview section:

Done, and I've cleaned out non-research elements from the milestones.

@Noc2 and @laboon:

> the price does seem rather expensive

A couple of things will help a lot:
 
- Could you point to grants you consider comparable? (I did seek guidance [here](https://forum.polkadot.network/t/tokenomics-grant-models/1266?u=taqtiqa-mark) but got no response)
- Could you indicate items/topics/milestones you think should be de-scoped.

I'll wait the other reviewer feedback to see where you all land before revising scope etc.

**Comment by jonasW3F** [comment]
Thanks for the application. After reading it, I struggle to see the clear scope and contribution of this study. As far as I understand, you intend to "[...] provide a creative synthesis of existing token-economy research" of the Polkadot parachains? How would that look like? You state that you do this by reviewing their whitepapers, and published/working papers. While many of the former exist, rigorous (academic) token economic research is mostly absent. From the existing whitepapers, most parts focus on the usecases and initial distribution schemes. The token economics is mostly limited to some arbitrarly defined inflation rate which is split between collators and potentially stakers (if existing). As another dimension of your analysis, you mention different types of tokens. I'd not expect much other tokens than utility tokens that pay for transaction fees, are used in governance for voting and/or might be used for staking. Other token types are virtually absent as they'd constitute clear securities. 

You also state that you will not be answering "[...] any particular token-economy design question". Further, I am also missing details on how you intend to analyse the situation with respect to the rational expectations equilibrium. The high price-tag and those open points make me question the contribution of the study and thereby how much it is needed.

In the case I've misinterpreted your approach, could you further elaborate?

**Review by dsm-w3f — COMMENTED** [review]
Thank you for the application. I'll describe below the points that raised my attention after reading it. Hope to hear from you about them soon. 

What will be the methodology planned for searching papers in the following categories?
 - Published articles	  
 - Working papers	   
 - Polkadot parachain whitepapers
If you gonna plan the methodology as part of the grant work, please include that as a deliverable. Is common to use systematic approaches for doing academic literature reviews, such as systematic reviews or systematic mappings. This increase the confidence in the review. Working papers and whitepapers are considered grey literature and could have another approach for searching. 

There is only one researcher doing all the work. How would you deal with researcher bias in the data collection/extraction for the papers in this case?

The current (incomplete) list of research to be investigated has some papers that are not from Polkadot ecosystem, such as Bitcoin. Why that? They will be used for comparison as related work? As for related work I think is fine. However, as papers to read and extract data for analysis make sure that your research as stated "A survey of some economic models used to develop Substrate (Dotsama) blockchain/token economies" include only Substrate (Dotsama) ecosystem. As far as I understood, the research is about already done Substrate (Dotsama) ecosystem tokenomics helping new projects with the knowledge of previous ones.  Correct me if I'm wrong. 

Deliverables:
We are developing an appropriate template for research grant submission. This is working in progress and should be ready soon. My apologies to have to deal with an inappropriate template for deliverables in a research grant. To summarize, the main deliverable for a research grant is the knowledge generated by the research, which should be transferred to the audience in some format, in your case a research paper. In this way, the final delivery from the grant should be the full paper available online somewhere (a preprint on arxiv.org would be enough). The deliverables in your application are most research questions. What would be expecting to receive in those deliverables? Text, tables, methodology, all of them? It is unclear to me the format of the deliverable and if all prerequisites to evaluate them will be present (methodology, raw data, analysis procedures, text describing the results). Usually, research projects are organized and developed in this order (could be different in your case): goals definition, methodology definition, execution, raw results extraction, data analysis, comparison with related work, complete the writing of the paper, and publishing. Could you please explain what will be the process that you will follow to perform the research and how they are time-based- related to your milestone deliveries planned? The milestones should be time-based in the order of delivery.

Research Profile, could you please provide a profile in at least one research indexer (Google Scholar, Researchgate, or another one)? This is just to have your publication record.

Regarding the price, I saw in your Linkedin profile that you are working for Westpac currently and the price for each delivery is 1 FTE. How would you deliver 1 FTE alone working in someplace else? Could elaborate more about the formation and rationale behind the price charged?

**Comment by taqtiqa-mark** [comment]
Thanks @jonasW3F and @dsm-w3f for the thoughtful feedback. I believe we are at 4 out of 5 reviewers?

@dsm-w3f thanks for the prompt to update my Linked-In status. If you could update your comment accordingly, I'd appreciate that. I'm currently looking for my next contract.

**Comment by taqtiqa-mark** [comment]
> @Noc2 and @laboon:
> 
> > the price does seem rather expensive
> 
> A couple of things will help a lot:
> 
>    * Could you point to grants you consider comparable? (I did seek guidance [here](https://forum.polkadot.network/t/tokenomics-grant-models/1266?u=taqtiqa-mark) but got no response)
 
As best I can infer/guess... the comparable you have in mind is [Milestone 1 of Uncollateralized Stablecoin Research](https://github.com/w3f/Grants-Program/blob/master/docs/rfps/Under%20Development/uncollateralized-stablecoin-research.md)

Correct?

**Comment by taqtiqa-mark** [comment]
Context to proposed budget: 
I assumed (correctly it appears) my day rate would be out of budget. I used the midpoint of the Glassdoor salary distribution for a Senior Economist (USA): USD 156k p.a.
I was originally [unable to find a related research topic](https://forum.polkadot.network/t/tokenomics-grant-models/1266). The only grant I was familiar with related to [re-implementation of prior on-chain IPFS grants @ 15K USD/month](https://github.com/w3f/Grants-Program/pull/1187#issuecomment-1283354955)
Since then I found [Milestone 1 of Uncollateralized Stablecoin Research](https://github.com/w3f/Grants-Program/blob/master/docs/rfps/Under%20Development/uncollateralized-stablecoin-research.md)

**Comment by taqtiqa-mark** [comment]
@dsm-w3f

> What will be the methodology planned for searching papers in the following categories?

I've added methodology to the Milestone 1 deliverables.

> Is common to use systematic approaches for doing academic literature reviews, such as systematic reviews or systematic mappings. This increase the confidence in the review. Working papers and whitepapers are considered grey literature and could have another approach for searching.

This exercise will be closest to a "Rapid Review", see Grant MJ, Booth A. A typology of reviews: an analysis of 14 review types and associated methodologies. Health Info Libr J. 2009 Jun;26(2):91-108. doi: 10.1111/j.1471-1842.2009.00848.x. Review. PubMed PMID: 19490148.

> There is only one researcher doing all the work. How would you deal with researcher bias in the data collection/extraction for the papers in this case?

I have adjusted the source of the research to eliminate mysely as a source of bias in the collection of research papers. 
The Polkadot networks are those Parachains elected as at 19 December 2022.


> We are developing an appropriate template for research grant submission.

That is great.  This application targets the current template requirements, since that is all that is known.

> This is working in progress and should be ready soon. My apologies to have to deal with an inappropriate template for deliverables in a research grant. To summarize, the main deliverable for a research grant is the knowledge generated by the research, which should be transferred to the audience in some format, in your case a research paper. In this way, the final delivery from the grant should be the full paper available online somewhere (a preprint on arxiv.org would be enough).

This was addressed under Future Plans:

> The working paper will be posted to [SSRN (e.g. FEN - Cryptocurrency Research eJournal)](https://papers.ssrn.com/sol3/JELJOUR_Results.cfm?form_name=journalBrowse&journal_id=3839004), [IDEAS](https://ideas.repec.org/), [Arxiv (q-fin.GN)](https://arxiv.org/list/q-fin.GN/recent)

I've repeated the detail in the final milestone.

> The deliverables in your application are most research questions. What would be expecting to receive in those deliverables? Text, tables, methodology, all of them? It is unclear to me the format of the deliverable and if all prerequisites to evaluate them will be present (methodology, raw data, analysis procedures, text describing the results). Usually, research projects are organized and developed in this order (could be different in your case): goals definition, methodology definition, execution, raw results extraction, data analysis, comparison with related work, complete the writing of the paper, and publishing. Could you please explain what will be the process that you will follow to perform the research and how they are time-based- related to your milestone deliveries planned? The milestones should be time-based in the order of delivery.

I've attempted to address these questions.

Please let me know if anything further is required.

> Researcher Profile, could you please provide a profile in at least one research indexer (Google Scholar, Researchgate, or another one)? This is just to have your publication record.

https://www.researchgate.net/profile/Mark-Van-De-Vyver-2

**Comment by taqtiqa-mark** [comment]
@jonasW3F

> Thanks for the application. After reading it, I struggle to see the clear scope and contribution of this study. As far as I understand, you intend to "[...] provide a creative synthesis of existing token-economy research" of the Polkadot parachains?

I've clarified this point.

> You state that you do this by reviewing their whitepapers, and published/working papers. While many of the former exist, rigorous (academic) token economic research is mostly absent.

The presence/absence of rigorous (academic) token economic research is the subject of this survey.
How many papers are required to say "rigorous (academic) token economic research is mostly" complete? This depends on your idea of 'rigor' and 'complete'. If you know of an existing survey establishing what you claim, could you please point it out?

> From the existing whitepapers, most parts focus on the usecases and initial distribution schemes. The token economics is mostly limited to some arbitrarly defined inflation rate which is split between collators and potentially stakers (if existing).

I currently share this assessment. But, I'm not persuaded this state-of-affairs is because richer ideas don't exist.

> As another dimension of your analysis, you mention different types of tokens. I'd not expect much other tokens than utility tokens that pay for transaction fees, are used in governance for voting and/or might be used for staking. 

As above.

> Other token types are virtually absent as they'd constitute clear securities.

Or are other token types virtually absent because they haven't been introduced?  Nonetheless, I've clarified that what is/not considered a security is out of scope.

> You also state that you will not be answering "[...] any particular token-economy design question". 

Correct, this is a survey, hence limited to what the existing literature addresses. Within the scope of the deliverables, I am not committing to solving any particular token-economy design question.

> Further, I am also missing details on how you intend to analyse the situation with respect to the rational expectations equilibrium. 

I use REE to delimit the scope of the genera token-economy research literature.  The REE framework naturally suggests some features of an economy, and I've now indicated where that is.  I had given an outline of the approach when justifying the exclusion of cryptocurrencies.

Let me know if the changes don't address a particular point to your satisfaction.

**Comment by dsm-w3f** [comment]
@taqtiqa-mark thank you for your answers. After we started this application review, we approved a [new template for research grants application](https://github.com/w3f/Grants-Program/blob/master/applications/application-template-research.md). Would you mind adjusting your application to this new template? I think there will be a few changes but especially in the deliverables section, we will have a better structure. Regarding your answers:

- Rapid Review is ok as a methodology since time is always a constraint. Just make sure that this doesn't affect the quality of your review and state the approach in the methodology. I think you will be able to find proper guidelines to perform this kind of review. 
- Researcher bias is more on how the researcher extracts data and analyses it since could be influenced by its background and beliefs than to set a fixed scope for data for analysis. Usually, research bias is reduced by having more than one researcher doing data analysis in an independent way and comparing the results. As this cannot be done when having only one researcher at least plan to declare this threat to the validity in the paper. 

For now, I'll wait for my colleagues to circle back to their comments. Let me know if you plan to adjust the application to the new template.

**Comment by taqtiqa-mark** [comment]
> For now, I'll wait for my colleagues to circle back to their comments. Let me know if you plan to adjust the application to the new template.

I'll also wait on their feedback before moving to the new template.

**Comment by Polkadot-Forum** [comment]
This pull request has been mentioned on **Polkadot Forum**. There might be relevant details there:

https://forum.polkadot.network/t/polkadot-the-centralized-decentralized-ecosystem/1412/7

**Review by Noc2 — CHANGES_REQUESTED** [review]
Sorry for the late response here. I missed the updates for some reason. First, I noticed that you changed the level to 2, but the application is still above 30k, see https://github.com/w3f/Grants-Program#rooster-level-3 Could you update this? I personally would still favor a smaller initial grant application for 30k or less. This way, we also get a better impression of your work and could potentially sign follow-up grants. Regarding the RFP, it was indeed already implemented a while ago, and we forgot to update it here. [I just fixed it](https://github.com/w3f/Grants-Program/commit/e5cdec3c074acc26abef007139ef9fdc16812a93). The price itself on the RFP was copy-pasted from another RFP. In the end, we paid the following for it: https://github.com/w3f/Grants-Program/blob/master/applications/stardust.md, and the team actually worked a lot more on it than only two weeks. The [on-chain IPFS grants @ 15K USD/month](https://github.com/w3f/Grants-Program/pull/1187#issuecomment-1283354955) was highly controversial internally and was only accepted because of the other work by TDSoftware. 

Regarding the research itself, I think one concern that we currently have is that researching the current state, especially for this amount, might not be so helpful. I agree that this might be the starting point, but ideally, as the next step (follow-up grant), potential solutions or improvements are researched/developed.

**Comment by keeganquigley** [comment]
Hi @taqtiqa-mark friendly reminder to look at the requested changes above. Thanks!

**Comment by taqtiqa-mark** [comment]
I've updated the application to use the new research template.
I've reduced the budget as requested, and update the scope accordingly.
Appreciate any feedback and suggestions.

**Review by Noc2 — DISMISSED** [review]
Thanks for the update. I'm happy to go ahead with it and mark it as ready for review.

File: applications/tokenomics-survey-2022.md
Comment (dsm-w3f) [inline applications/tokenomics-survey-2022.md:301]: Methodology as an appendix is unusual for academic papers, at least in computer science research, area that I'm used to read papers. Is this scientific/academic research or an informal one? For academic research, the methodology is important and provides information to check the validity of the findings, thus usually is a section in the body of the article. Another unusual thing is that I could not find a related work section. What is the intended profile of the reader of this article? Is that a scientific/academic or informal one? Besides the preprint, what is the intended venue of publication? They accept this proposed structure of the paper?

File: applications/tokenomics-survey-2022.md
Comment (dsm-w3f) [inline applications/tokenomics-survey-2022.md:302]: We are gently asking our applicants/future grantees that add an acknowledgments section to the end of the article with the statement: "This work was supported by a research grant from the Web3 Foundation.". Could you please add it to your deliverables?

**Review by dsm-w3f — CHANGES_REQUESTED** [review]
I added some comments/requests in the body of the application. I think now the proposal is a more informal paper (working paper). Is that right? I made some comments considering the formalism of research papers. If the focus of the paper has changed, let me know and some of my comments regarding methodology could not be applicable. There are other comments that need to be fixed anyway.

**Comment by taqtiqa-mark** [comment]
> I added some comments/requests in the body of the application. I think now the proposal is a more informal paper (working paper). Is that right? I made some comments considering the formalism of research papers. If the focus of the paper has changed, let me know and some of my comments regarding methodology could not be applicable. There are other comments that need to be fixed anyway.

@dsm-w3f , thanks for the detailed feedback. It will help if you can add **Blocking:** before each detail that is blocking approval, and break it into a separate paragraph.

Re the search - I'm surprised you had access, unless you are a resident of NSW? To confirm, they took about a week to respond? That was the time frame I was advised.  Did they indicate which of the databases they searched?

**Comment by dsm-w3f** [comment]
> It will help if you can add **Blocking:** before each detail that is blocking approval, and break it into a separate paragraph.

I expect that all be resolved/answered before my approval.


> Re the search - I'm surprised you had access, unless you are a resident of NSW? To confirm, they took about a week to respond? That was the time frame I was advised. Did they indicate which of the databases they searched?

I just used the link that you posted. For me is just a search engine, Is it something different from that? Why use a service that is not freely available to anyone? Any advantage to using this service?

File: applications/tokenomics-survey-2022.md
Comment (taqtiqa-mark) [reply applications/tokenomics-survey-2022.md:300]: > Is this scientific/academic research or an informal one?

The target audiences are not academic.

> What is the intended profile of the reader of this article?

That is in the **Ecosystem Fit** section.

> what is the intended venue of publication?

The pre-print archives have been provided.  The original proposal was suitable for subsequent publication, but that was assessed as too risky and expensive - this propsal has been revised as requested previously.

File: applications/tokenomics-survey-2022.md
Comment (taqtiqa-mark) [reply applications/tokenomics-survey-2022.md:296]: The template states these are mandatory.

File: applications/tokenomics-survey-2022.md
Comment (taqtiqa-mark) [reply applications/tokenomics-survey-2022.md:265]: The template states these are mandatory.

File: applications/tokenomics-survey-2022.md
Comment (taqtiqa-mark) [reply applications/tokenomics-survey-2022.md:206]: > I don't think that the ...

As you notice these choices are driven by ... researcher bias (per you're earlier feedback). To my mind what you are calling 'bias' is, in this context, actually skill and experience. However, it was also correctly pointed out that the assessors have no idea about what that skill/experience level is. Hence the direction to bring the budget down.

@Noc2 is the article selection really going to be micro-managed in the proposal stage by a reviewer who, in their own words asked that individual bias be removed where possible:

> Researcher bias is more on how the researcher extracts data ...

I've moved the list of articles to be reviewed into it's own deliverable under Milestone 1.
My understanding is that you can withhold a payment or terminate the grant if the deliverables are as low quality as @dsm-w3f is expecting.

> Another option is to remove academic papers from the scope of the review since you will probably not find anyone regarding polkadot/kusama parachains tokenomics.

This has been addressed:

> This project provides a annotated bibliography to help think through various choices and their trade-offs. It does so by reviewing the current state of play, as it is described in: a) published general-blockchain research, b) Polkadot whitepapers.

> [This one](https://jbba.scholasticahq.com/article/57668.pdf) is worthwhile to take a look.

Thanks for the reference. If that article meets the selection criteria it will be reviewed.

**Comment by taqtiqa-mark** [comment]
> I expect that all be resolved/answered before my approval.

I've resolved all the issues I can. I've asked @Noc2 to resolve the only open issue - unless you do so you're self.

>  Why use a service that is not freely available to anyone? 

This is the debate about the subscription vs surveillance business models. 

> Any advantage to using this service?

These databases provide impact factors which are used in the selection criteria you have been reading for some time now.  While flawed they are useful as an initial quality filter that is not biased against the newest research.

File: applications/tokenomics-survey-2022.md
Comment (dsm-w3f) [reply applications/tokenomics-survey-2022.md:206]: @taqtiqa-mark let me explain again my concern about researcher bias quoting the full statement that did before and not only one part of it.

> * Researcher bias is more on how the researcher extracts data and analyses it since could be influenced by its background and beliefs than to set a fixed scope for data for analysis. Usually, research bias is reduced by having more than one researcher doing data analysis in an independent way and comparing the results. As this cannot be done when having only one researcher at least plan to declare this threat to the validity in the paper.

If you want to know more about researcher bias a good reference is [this one](https://doi.org/10.1002/9781119547167.ch76). For knowing more about academic systematic review and the rigor of it I suggest the [seminal work of Kitchenham](https://www.researchgate.net/publication/302924724_Guidelines_for_performing_Systematic_Literature_Reviews_in_Software_Engineering). 

You proposal for trying to solve this bias does not solves the problem and creates another one, as I stated before. Even you commissioned a service to search for literature, there are several other steps in the literature review that will be performed only by yourself, being subject to researcher bias.

> I don't think that the selection of a New South Wales information search service for doing research on published articles is a good idea. This does not reduce bias, in fact, adds a selection bias since the service is probably less complete than other ones. There are few research articles published regarding this specific theme. I used the service cited and did not find any relevant article. I used Google Scholar with the query string ("blockchain" + "token" + "economics" + "rational expectations equilibrium") and also found almost nothing strongly related to the theme (google scholar returned 17 papers in total). I think you can use a more broad search engine for doing this research (like Google Scholar or Scopus) and I bet your scope of review is still small. Another advantage is that you can argue that you have a more strong methodology to try to publish the paper in a good venue. In fact, most of the papers returned in my quick search are more to a related work section than to compose the body of the literature review.

As you [stated in the answer to another question](https://github.com/w3f/Grants-Program/pull/1348#discussion_r1091150144), this is not academic/scientific research anymore. So, this rigor is not needed.

File: applications/tokenomics-survey-2022.md
Comment (dsm-w3f) [reply applications/tokenomics-survey-2022.md]: image

I mean this part between 0e. and 0e.1.

File: applications/tokenomics-survey-2022.md
Comment (dsm-w3f) [reply applications/tokenomics-survey-2022.md]: Same as the comment above.

File: applications/tokenomics-survey-2022.md
Comment (taqtiqa-mark) [reply applications/tokenomics-survey-2022.md:206]: Apologies for any misunderstanding, but I am confused that you haven't resolved this comment.
Can you please state what you would like to be done?

Comment (dsm-w3f) [reply applications/tokenomics-survey-2022.md:206]: For me is resolved.

**Review by dsm-w3f — APPROVED** [review]
I'm happy to support you application.

**Review by Noc2 — APPROVED** [review]

**Comment by taqtiqa-mark** [comment]
I expect a delay of a week or two wrt Milestone 1:

It has taken a while to disentangle what blockchain token economy features arise from non-discretionary chain properties (Substrate requirements) and what are discretionary (Polkadot implementation choices).  

The good news is that, so far, the problematic features are discretionary and can be elided wlog. The bad news is the descriptions will have to be by reference to Substrate - otherwise the text will read as criticisms/refutation of claims that DOT is not a security, and distract from the objective - bringing to light tools/technology applicable to designing a token-economy.

**Pull request merged**
